lib/public_key/src/pubkey_cert.erl

```diff
@@ -33,7 +33,8 @@
 	 is_fixed_dh_cert/1, verify_data/1, verify_fun/4, 
 	 select_extension/2, match_name/3,
 	 extensions_list/1, cert_auth_key_id/1, time_str_2_gregorian_sec/1,
-         gen_test_certs/1, root_cert/2]).
```

Comment:
I would remove this change as it only changes whitespaces in a non-related module.

Reply:
ok